packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -105,7 +107,9 @@ export const CustomScrollbar: FC<Props> = ({
     <Scrollbars
       ref={ref}
       className={classNames(styles.customScrollbar, className)}
-      onScroll={setScrollTop}
+      onScrollStop={() => {
+        ref.current && setScrollTop && setScrollTop(ref.current.getValues());
+      }}
```

Comment:
this will create a new callback every render, even when setScrollTop is not passed from parent, could we memoize this callback with useCallback?

Reply:
Updated